elasticsearch/src/main/java/org/hibernate/search/elasticsearch/analyzer/impl/ElasticsearchAnalyzerStrategy.java

```diff
@@ -7,12 +7,14 @@
 package org.hibernate.search.elasticsearch.analyzer.impl;
 
 import java.util.Collection;
+import java.util.Collections;
```

Comment:
Mmmmh, the commit message does not really state the content of this commit. It's far more extensive AFAICS.